solr/solr-ref-guide/src/searching.adoc

```diff
@@ -27,7 +27,8 @@
   realtime-get, +
   exporting-result-sets, +
   parallel-sql-interface, +
-  analytics
+  analytics, +
```

Comment:
You've placed this new page at the end here but it feels more important than that.  Any way I have no real suggestion here on where to place it; there are _many_ pages under Search and there isn't much of an organization.